CIP-0088/README.md

```diff
@@ -0,0 +1,556 @@
+---
+CIP: 88
+Title: Token Policy Registration
+Category: Tokens
+Status: Proposed
+Authors:
+- Adam Dean <adam@crypto2099.io>
+Implementors:
+-
```

Comment:
```suggestion
Implementors: NA
```
... just noticed in the meantime (apologies for quoting set theory) that this is not a null set of implementors: it's a set with a single null implementor 🤓
...

Reply:
This was a temporary solution in the interim as I intend to list working tools and interfaces here on the "Path to Active" :) Can change to an empty array for the time being if it makes merging easier

Reply:
We just can't merge it as-is because it needs to both be valid YAML (which it is already) but also not have a blank implementor... which wouldn't break the site builders but would look like something was off in the metadata. 🤔